regulations/pages/history.md

```diff
@@ -3,13 +3,13 @@
 ## Older Versions of the Regulations
 
 Until 2011, the Regulations were maintained by Ron van Bruchem and the WCA Board. Since then, the [WCA Regulations Committee](https://www.worldcubeassociation.org/contact/wrc) is in charge of them.  
-For the 2013 release, the Regulations were split into two documents: the Regulations and the Guidelines.  
-Starting with 2013, updates (with lists of changes changes) will be announced on the [WRC Announcements page](https://www.worldcubeassociation.org/regulations/announcements/).
+For the 2013 release, the Regulations were split into two documents: the Regulations and the Guidelines.
```

Comment:
No idea why this is displayed this way, I just removed the link to the announcement page because it doesn't exist anymore (#97).